Source/Tests/Paths/ConfigDirPathTests.cs

```diff
@@ -2,25 +2,36 @@ namespace CodeImp.Bloodmasters.Tests.Paths;
 
 public class ConfigDirPathTests
 {
-    [Fact(DisplayName = "ConfigDirPath should be named 'Debug' in dev mode and 'Config' otherwise")]
-    public void ConfigDirPathShouldBeNamedDebugInDevModeAndConfigOtherwise()
+    [Fact(DisplayName = "ConfigDirPath should be named 'Config' in production mode")]
+    public void ConfigDirPathShouldBeNamedConfigInProductionMode()
     {
         // Arrange
-        var dirName = Path.GetFileName(CodeImp.Bloodmasters.Paths.ConfigDirPath);
+        var sut = CodeImp.Bloodmasters.Paths.Instance;
+        var dirName = Path.GetFileName(sut.ConfigDirPath);
 
         // Assert
-        Assert.Equal(
-            CodeImp.Bloodmasters.Paths.IsDevModeBuild ? "Debug" : "Config",
-            dirName);
+        Assert.Equal("Config", dirName);
```

Comment:
Let's assert that the current mode is `Production` as well?

Reply:
We can create a separate test to check the current mode depending on the passed `StartupMode` so that we don't have to check this in other tests

Reply:
Please, check https://github.com/ForNeVeR/bloodmasters/pull/149/commits/50b0f2018ddba69b138a5d8363ac6fe548077912